debezium-core/src/main/java/io/debezium/transforms/ScriptingTransformation.java

```diff
@@ -93,13 +95,21 @@ public static NullHandling parse(String value, String defaultValue) {
         }
     }
 
+    private static final Field TOPIC_REGEX = Field.create("topic.regex")
```

Comment:
@gunnarmorling Could we have a list of regexes?

Reply:
What's the advantage over simply using "foo|bar"?

Reply:
Was there anything else left to discuss on this point?

Reply:
I think that not, if you agree that giving multiple patterns with the pipe is good enough. I'd argue it's even better, as comma-separated values will require escaping.

Reply:
In other words, feel free to merge it, @Naros :)